lit_gpt/lora.py

```diff
@@ -470,22 +470,19 @@ def __init__(self, config: Config) -> None:
         assert config.padded_vocab_size is not None
         self.config = config
 
-        if config.to_head:
-            self.lm_head = LoRALinear(
-                config.n_embd,
-                config.padded_vocab_size,
-                bias=False,
-                r=config.r,
-                lora_alpha=config.alpha,
-                lora_dropout=config.dropout,
-            )
-        else:
-            self.lm_head = nn.Linear(config.n_embd, config.padded_vocab_size, bias=False)
+        self.lm_head = LoRALinear(
+            config.n_embd,
+            config.padded_vocab_size,
+            bias=False,
+            r=config.r if config.to_head else 0,
```

Comment:
```suggestion
            r=(config.r if config.to_head else 0),
```

Reply:
I'm not against it, just wondering whether it's up to some code convention or not?